templates/docs/examples/patterns/suru/divider.html

```diff
@@ -0,0 +1,25 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Suru / Divider{% endblock %}
+
+{% block standalone_css %}patterns_suru{% endblock %}
+
+{% set is_paper = True %}
+{% block content %}
+<div class="p-suru--divider">
+    <div class="row--25-75">
```

Comment:
This may even be a question, why are you using `row--25-75` if there is only one column? Shouldn't this not even use grid?

Reply:
Surus are usually used with hero. And hero by default should align with 75% column - not start from the start of the row. And this will happen if you use 25/75 with just one column.

But you are right, in the simplest case, if we don't want any layout, no grid would be needed.